core/templates/dev/head/pages/subtopic-viewer-page/subtopic-viewer-page.directive.html

```diff
@@ -0,0 +1,42 @@
+<background-banner></background-banner>
+<div class="oppia-subtopic-viewer-container" style="position: relative;">
+  <div class="oppia-subtopic-title">
+    <[$ctrl.subtopicTitle]>
+  </div>
+  <div class="oppia-page-contents-viewer">
+    <div>
+      <angular-html-bind html-data="$ctrl.pageContents.getHtml()">
+      </angular-html-bind>
+    </div>
+  </div>
+</div>
+<style>
+.oppia-subtopic-viewer-container .oppia-subtopic-title {
```

Comment:
Indent following lines (in style tag) by 2.

Reply:
Done